src/Controller/AdminUserController.php

```diff
@@ -46,7 +46,7 @@ public function userActivate(User $user): Response
     {
         $active = $user->getActivate();
         $user->setActivate(!$active);
-        if ($active == true) {
+        if ($active == false) {
```

Comment:
tu fais le message à chaque fois, donc tu peux faire plutot qq chose comme 
$activeMessage = 'activé';
if ($user->getActivate()) {
    $activeMessage = 'desactivé":
}
  $this->addFlash('success','L'utilisateur à bien été '. $activeMessage . '!');